build.gradle

```diff
@@ -1,7 +1,6 @@
 plugins {
   id 'application'
   id 'checkstyle'
-  id 'pmd'
```

Comment:
We are only using checkstyle in most of the reform projects, as there is overlap between both of these tools. I also saw this being removed in one of our existing PR in this project, so thought we are getting rid of it. Please advise otherwise.

Reply:
I don't really have a strong opinion on it, if it doesn't provide much value then feel free to remove it. We didn't use it in previous project either. Resolving the conversation.